### ISSUE TITLE
Non voters counter

### ISSUE THREAD

**Issue by bissim** [Wolfsblvt/advancedpolls#88]
Nullifying the vote by looking at poll results is good, yet wouldn't be possible to show a **_non voters_ counter** and even their nicknames if the _show voters_ option is enabled?

**Comment by javiexin** [comment]
I don't understand what you are asking for.
A list of all forum members that have not voted on a poll?  Forget it...  And the count is kind of useless.
If this is not what you mean, please, explain it better.

Maybe an option to see who voted without seeing what they voted for would make sense, but I doubt it as well  (cost/benefit would be low, imho).

**Comment by javiexin** [comment]
Ah! Now I think I understood what you want! To see who opted to not issue a vote, yet see the results, right?  Well, that would be possible, I think.  Let me see how complicated that would be.  No promise!

**Comment by Mauron** [comment]
Right, a counter (and possibly a list) of people who clicked on *I don't want to vote, show results*.

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using the project. This project is no longer maintained, so I’m closing all remaining issues as part of archiving the repository. This avoids leaving a false impression that further development or support is planned.

Anyone interested in continuing development is welcome to do so through a fork, subject to the repository’s license and any applicable third-party licenses.

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using the project. This repository is no longer maintained and is being archived, so I’m closing all remaining issues rather than leaving them open with the implication of future development or support. See the [Disclaimer](https://github.com/Wolfsblvt/advancedpolls#disclaimer) for details.
